src/pages/react/unit-testing-fundamentals-explained-using-javascript.md

```diff
@@ -6,6 +6,7 @@ author: alex
 imageUrl: https://firebasestorage.googleapis.com/v0/b/reactjsacademy-react.appspot.com/o/blog%20post%20images%2Fside-effects-min.png?alt=media
 authorTwitter: alex_lobera
 order: 8
+tags: ['react', 'beginner', 'testing', 'javascript']
```

Comment:
I guess we need to go through and add tags to all the posts? I was also wondering if we still need the "order" in the frontmatter, I suppose it still is relevant for the main blog page but as we increase the number of posts it makes sense to use this taxonomy so they are grouped by subject when they appear.

Reply:
yeah, we need to tag all the content. I'd do that in a different PR thought. Yes we need the order for the blog page